rskj-core/src/test/resources/dsl/transaction_storage_rskip446/dynamic_reentrancy_context_revert_or_invalid_undoes_all.txt

```diff
@@ -0,0 +1,154 @@
+comment
+
+// CONTRACT CODE
+
+pragma solidity ^0.8.24;
+
+contract TstorageDynamicReentrancyContext {
+    bool reentrant = false;
+
+    constructor() {
+    }
+
+    event OK();
+    event ERROR(string, uint256);
+
+    function opcodeUndoesAll(uint256 opcodeSelector) external {
```

Comment:
praise: I really like the way the test is structured, and the function names. <3